canonical-web-and-design/canonical.com
# Redesigned /partners index

## Done

- Updated /partners to match the design
- Updated top navigation to use the new logo, search and a dropdown for /partners pages

## QA

- View the site in your web browser at: https://canonical-com-521.demos.haus/partners
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See that the page matches the [design](https://app.zeplin.io/project/624ed20e0af92866116ea26d/screen/625418358b950467d5a7f9e2)

## Issue / Card

Fixes [#5076](https://github.com/canonical-web-and-design/web-squad/issues/5076)
